api-put-object.go

```diff
@@ -35,6 +35,7 @@ import (
 // PutObjectOptions represents options specified by user for PutObject call
 type PutObjectOptions struct {
 	UserMetadata            map[string]string
+	ObjectTags              map[string]string
```

Comment:
We can call it UserTags? @nitisht because there is a concept of private tags in S3 which makes this userSet tags.